src/halmos/__main__.py

```diff
@@ -819,19 +836,25 @@ def main() -> int:
     # compile
     #
 
-    try:
-        crytic_compile_args, crytic_compile_unknown_args  = crytic_compile_parser.parse_known_args()
+    build_cmd = [
+        'forge', # shutil.which('forge')
```

Comment:
not sure if we need `shutil.which('forge')` here, or not, especially in the windows env. what do you think, karma?

Reply:
I don't think so, this should be fine